src/main/java/io/jawg/osmcontributor/flickr/util/FlickrPhotoUtils.java

```diff
@@ -56,4 +58,29 @@ public static RestAdapter getAdapter() {
         }
         return adapter;
     }
+
+    public static RestAdapter getAdapter(final Map<String, String> oAuthParams) {
+        RestAdapter adapterOauth = null;
+        OkHttpClient okHttpClient = new OkHttpClient();
+        try {
+            SSLSocketFactory NoSSLv3Factory = new NoSSLv3SocketFactory(new URL("https://api.flickr.com/services"));
+            okHttpClient.setSslSocketFactory(NoSSLv3Factory);
+            adapterOauth = new RestAdapter.Builder()
+                    .setConverter(new StringConverter())
+                    .setEndpoint("https://api.flickr.com/services")
+                    .setClient(new OkClient(okHttpClient)).setRequestInterceptor(new RequestInterceptor() {
+                        @Override
+                        public void intercept(RequestFacade request) {
+                            request.addHeader("Authorization", FlickrSecurityUtils.getAuthorizationHeader(oAuthParams));
+                        }
+                    })
+                    .setLogLevel(RestAdapter.LogLevel.FULL).setLog(new AndroidLog("---------------------->"))
```

Comment:
LogLevel should not be too verbose in production... Either set it to a more reasonable level to avoid overflowing the android syslog, or use the Android DebugConfig variable to take it away only for releases:
```java
if (BuildConfig.DEBUG) {
  // do something for a debug build
}
```